server/lib/enrich/article.js

```diff
@@ -25,6 +25,10 @@ module.exports = exports = function article(content, format) {
 		content.content_type = CONTENT_TYPE_ALIAS[content.type] || content.type;
 	}
 
+	if (content.contentStats && content.contentStats.graphics && content.contentStats.graphics > 0){
+		content.content_type = 'rich article'
+	}
+
```

Comment:
This could introduce data integrity issues and it would break other parts of `next-syndication-api` that rely on `content_type === 'article'`. 

This logic would better reside in the [export.js](https://github.com/Financial-Times/next-syndication-api/blob/master/server/controllers/export.js#L32-L34) controller (At the point of creating the csv file).

Reply:
Yes, it would be nice, but what that export does is reads the database and copy-pastes it into the csv file. 
Because of that we need to make sure that either 1) the database saves the count of rich content in a separate column or 2) (preferred, in my opinion) when we are sending stuff to the database, it can have a type of `rich article`

Reply:
Either way we'd need to change the database schema (the content types are an `enum` so that would need to be updated). The alternative - would be to ping Elastic search for each article we're trying to export to figure out if it's a rich article - that feels more "energy intensive".